web/src/main/webapp/WEB-INF/classes/log4j.xml

```diff
@@ -137,7 +137,7 @@
     <level value="ERROR"/>
   </logger>
   <logger name="geonetwork.harvest.wfs.features">
-    <level value="ERROR"/>
+    <level value="DEBUG"/>
```

Comment:
Should we be defaulting to debug level?

Reply:
During testing, I was checking the WFS feature indexing process (https://geonetwork-opensource.org/manuals/4.0.x/en/user-guide/analyzing/data.html) on the default samples Rivers basins of Africa and Physiographic map and it is working fine but as we don't have progress indicator, I turned it on indeed and people using that feature check the log quite often as source layers changed and when running regularly at some point errors happen (eg. invalid geom, network failures). Restored ERROR level.